templates/commands/render.go

```diff
@@ -322,6 +328,35 @@ func (r *Render) realRun(ctx context.Context, rp *runParams) (outErr error) {
 	return nil
 }
 
+// validateAndDefaultInputs validates and sets the input values provided by the user.
+func validateAndDefaultInputs(flagInputs map[string]string, spec *model.Spec) error {
```

Comment:
Can we also validate that no superfluous extra inputs were provided by the user?

Reply:
+1